mailbox/api/src/main/java/org/apache/james/mailbox/model/PartContentDescriptor.java

```diff
@@ -19,45 +19,49 @@
 
 package org.apache.james.mailbox.model;
 
-import static org.apache.james.mailbox.model.FetchGroup.NO_MASK;
-
+import java.util.EnumSet;
 import java.util.Objects;
 
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableSet;
+
 /**
  * Describes the contents to be fetched for a mail part. All
  * implementations MUST implement equals. Two implementations are equal
  * if and only if their paths are equal.
  */
 public class PartContentDescriptor {
```

Comment:
most of this class is copy/paste from FetchGroup, can't we find a way to reuse FetchGroup in PartContentDescriptor?

Reply:
Gonna investigate